UnityProject/Assets/Scripts/PlayGroups/PlayerSync.Server.cs

```diff
@@ -626,6 +626,28 @@ private void CheckSpaceDamage()
 		}
 	}
 
+	private bool AtSpace() //Checks if player is at space, and if it is, enables oxygen alert if suit is not present.
+	{
+		if (MatrixManager.IsSpaceAt(Vector3Int.RoundToInt(serverState.WorldPosition)))
+		{
+			if(!IsEvaCompatible())
+			{
+				UIManager.Instance.GetComponentInChildren<UI_OxygenAlert>().gameObject.GetComponent<UnityEngine.UI.Image>().enabled = true;
```

Comment:
Why not store a reference to this image somewhere in UIManager in Start() and access it instead of doing lookups every time?